<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Automator</class>
 <widget class="QMainWindow" name="Automator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Automator</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QWidget" name="presetBar" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QToolButton" name="togglePreset">
           <property name="icon">
            <iconset resource="automator.qrc">
             <normaloff>:/icons/resources/Collapse.png</normaloff>:/icons/resources/Collapse.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Preset</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="presetWidget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Preset</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QWidget" name="widget_4" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>8</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="saveas"/>
            </item>
            <item>
             <widget class="QLabel" name="filename">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Save to:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QWidget" name="widget_2" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QComboBox" name="saveto">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>4</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QComboBox" name="preset"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Save as:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="filedropBar" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QToolButton" name="toggleFiledrop">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="automator.qrc">
             <normaloff>:/icons/resources/Collapse.png</normaloff>:/icons/resources/Collapse.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>File drop</string>
           </property>
           <property name="icon" stdset="0">
            <iconset>
             <normaloff>:/icons/resources/Pin.png</normaloff>:/icons/resources/Pin.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="filedropWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QWidget" name="filedrop" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>12</number>
            </property>
            <property name="rightMargin">
             <number>16</number>
            </property>
            <item>
             <widget class="QProgressBar" name="progressBar">
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_2">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>430</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="displayValues">
     <property name="title">
      <string>Display values</string>
     </property>
     <addaction name="as8bitValues"/>
     <addaction name="as10bitValues"/>
     <addaction name="separator"/>
     <addaction name="asFloatValues"/>
     <addaction name="asHexadecimalValues"/>
     <addaction name="asPercentageValues"/>
    </widget>
    <widget class="QMenu" name="magnify">
     <property name="title">
      <string>Magnification</string>
     </property>
     <addaction name="magnify1x"/>
     <addaction name="magnify2x"/>
     <addaction name="magnify3x"/>
     <addaction name="magnify4x"/>
     <addaction name="magnify5x"/>
    </widget>
    <addaction name="active"/>
    <addaction name="pin"/>
    <addaction name="separator"/>
    <addaction name="displayValues"/>
    <addaction name="separator"/>
    <addaction name="magnify"/>
    <addaction name="separator"/>
    <addaction name="toggleMouseLocation"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="pick"/>
    <addaction name="drag"/>
    <addaction name="clear"/>
    <addaction name="separator"/>
    <addaction name="copyRGBAsText"/>
    <addaction name="copyHSVAsText"/>
    <addaction name="copyHSLAsText"/>
    <addaction name="copyHexAsText"/>
    <addaction name="copyIccAsText"/>
    <addaction name="separator"/>
    <addaction name="copyColorAsBitmap"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="about"/>
    <addaction name="openGithubReadme"/>
    <addaction name="openGithubIssues"/>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="viewMenu"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="copyRGBAsText">
   <property name="text">
    <string>Copy RGB as text</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="copyHSVAsText">
   <property name="text">
    <string>Copy HSV as text</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="copyColorAsBitmap">
   <property name="text">
    <string>Copy Color as bitmap</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="active">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/resources/Active.png</normaloff>:/icons/resources/Active.png</iconset>
   </property>
   <property name="text">
    <string>Active</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="toggleMouseLocation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show mouse location</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="copyIccAsText">
   <property name="text">
    <string>Copy ICC profile as text</string>
   </property>
  </action>
  <action name="as8bitValues">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>as 8-bit</string>
   </property>
   <property name="toolTip">
    <string>Display as 8-bit</string>
   </property>
  </action>
  <action name="as10bitValues">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>as 10-bit</string>
   </property>
   <property name="toolTip">
    <string>Display as 10-bit</string>
   </property>
  </action>
  <action name="asFloatValues">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>as Float</string>
   </property>
   <property name="toolTip">
    <string>Display as Decimal</string>
   </property>
  </action>
  <action name="asHexadecimalValues">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>as Hexadecimal</string>
   </property>
   <property name="toolTip">
    <string>Display as Hexadecimal</string>
   </property>
  </action>
  <action name="asPercentageValues">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>as Percentage</string>
   </property>
   <property name="toolTip">
    <string>Display as Percentage</string>
   </property>
  </action>
  <action name="magnify1x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1x</string>
   </property>
   <property name="toolTip">
    <string>Zoom 100%</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="magnify2x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2x</string>
   </property>
   <property name="toolTip">
    <string>Zoom 200%</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="magnify3x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3x</string>
   </property>
   <property name="toolTip">
    <string>Zoom 400%</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="magnify4x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4x</string>
   </property>
   <property name="toolTip">
    <string>Zoom 800%</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </action>
  <action name="magnify5x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5x</string>
   </property>
   <property name="toolTip">
    <string>Zoom 1600%</string>
   </property>
   <property name="shortcut">
    <string>5</string>
   </property>
  </action>
  <action name="openGithubReadme">
   <property name="text">
    <string>Open Github README</string>
   </property>
   <property name="toolTip">
    <string>Open Github README</string>
   </property>
   <property name="shortcut">
    <string>O</string>
   </property>
  </action>
  <action name="openGithubIssues">
   <property name="text">
    <string>Open Github Issues</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="pick">
   <property name="text">
    <string>Capture from pick</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="pin">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/resources/Pin.png</normaloff>:/icons/resources/Pin.png</iconset>
   </property>
   <property name="text">
    <string>Pin</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="clear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="copyHSLAsText">
   <property name="text">
    <string>Copy HSL as text</string>
   </property>
  </action>
  <action name="copyHexAsText">
   <property name="text">
    <string>Copy HEX as text</string>
   </property>
  </action>
  <action name="drag">
   <property name="text">
    <string>Capture from drag</string>
   </property>
   <property name="shortcut">
    <string>Shift+Return</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="automator.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>false</bool>
  </property>
  <property name="gridSnapY">
   <bool>false</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
